docs: Add a draft Readme to CMS (FI-16)

elien2016 opened the pull request:
## Describe your changes
updated README

## Issue ticket number and link
FI-16
https://catting.atlassian.net/browse/FI-16?atlOrigin=eyJpIjoiYWJlZTgxODJkYTBhNDY3ZGExMmZmZTViYzc2MmQ5MmMiLCJwIjoiaiJ9

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

ybw0014 commented:
We have updated the secret names (https://github.com/Charlie059/Flow.IO/pull/7#issue-2061112646), so it is better to update them in the sample as well.
I think we can add a `.env.example` under `@core/cms` so it wont take up space in readme.

Charlie059 commented:
@ybw0014, @elien2016 Thank you all!

elien2016 commented:
> We have updated the secret names ([#7 (comment)](https://github.com/Charlie059/Flow.IO/pull/7#issue-2061112646)), so it is better to update them in the sample as well. I think we can add a `.env.example` under `@core/cms` so it wont take up space in readme.

Updated. Lmk if there's anything else that needs changing:)

Charlie059 commented on @core/cms/.env.example:
Not sure if this can pass gitguardian, but let me try
